### ISSUE TITLE
"Build failed for unknown reason. See output log."

### ISSUE THREAD

**Issue by OvermindDL1** [JakeBecker/ide-elixir#4]
I keep getting this with my big project:

![image](https://user-images.githubusercontent.com/401605/33783999-567c1b58-dc1c-11e7-9693-008a18f9911a.png)

Running 1.6.0-dev master.

Manually running `mix compile` and whatever else all work fine.

Unable to find any semblance of an 'output log'...

**Comment by JakeBecker** [comment]
Sorry, Atom IDE silently discards log messages by default. Set `core.debugLSP` to `true` and it'll log them in the console, though you unfortunately have to expand each message individually to read them.

Alternatively, give it a try in the VS Code ElixirLS plugin which prints the log to the "Output" pane. The behavior will likely be the same, and it's easier to debug because of the better logging.

**Comment by OvermindDL1** [comment]
Managed to fix it before I got to this point, but updating elixir's master to HEAD (I was back a few weeks), I'm guessing something broke at some point because it was working earlier with that elixir version, some kind of failed check perhaps?

I guess this can be closed since it is the elixir dev version anyway, or left open to better error messages and such, up to you.  ^.^

And VSCode is not really an option here.
